tests/src/core/testqgis.cpp

```diff
@@ -136,5 +137,17 @@ void TestQGis::doubleToString()
   QCOMPARE( qgsDoubleToString( 12345, -1 ), QString( "12345" ) );
 }
 
+void TestQGis::qgsround()
+{
+  QCOMPARE( qgsRound( 3.141592653589793 ), 3. );
+  QCOMPARE( qgsRound( 2.718281828459045 ), 3. );
+  QCOMPARE( qgsRound( -3.141592653589793 ), -3. );
+  QCOMPARE( qgsRound( -2.718281828459045 ), -3. );
+  QCOMPARE( qgsRound( 314159265358979.3 ), 314159265358979. );
+  QCOMPARE( qgsRound( 2718281828459.045 ), 2718281828459. );
+  QCOMPARE( qgsRound( -314159265358979.3 ), -314159265358979. );
+  QCOMPARE( qgsRound( -2718281828459.045 ), -2718281828459. );
+}
```

Comment:
Maybe a couple of simple ["rounding halfway cases away from zero"](http://en.cppreference.com/w/cpp/numeric/math/round) tests? Just to make sure no one accidentally borks the implementation later on. e.g.

```C++
QCOMPARE( qgsRound( 1.5 ), 2. );
QCOMPARE( qgsRound( -1.5 ), -2. );
```